unified-installer/astra-unified-installer.sh

```diff
@@ -188,6 +209,90 @@ get_configs() {
     CONNECTOR_AUTOSUPPORT_URL="${CONNECTOR_AUTOSUPPORT_URL:-$__PRODUCTION_AUTOSUPPORT_URL}"
 }
 
+print_help() {
+    cat <<EOF
+Here is a list of available options and environment variables for configuring the Astra cluster installation script.
+Values which are required and have no default value are marked with '*'.
+
+Options:
+  --help/-h    Show this message and exit.
+  --required   Prints all required environment variables which do not have a default value.
+
+Environment Variables:
+  ----- Script Functionality
+  CONFIG_FILE                        Path to a configuration file. Overrides environment variables specified via command line.
+  DRY_RUN                            Skips applying generated resources when set to true. Default is false.
+  SKIP_IMAGE_CHECK                   Skips checking if images exist when set to true. Default is false.
+  SKIP_ASTRA_CHECK                   Skips all checks requiring a connection to Astra Control when set to true. Default is false.
+  SKIP_TLS_VALIDATION                Skips TLS validation for all requests to Astra Control, including the Connector (unless CONNECTOR_SKIP_TLS_VALIDATION is set) when set to true. Default is false.
+  DISABLE_PROMPTS                    Skips all prompts, answering 'yes' by default when set to true. Default is false.
+  DO_NOT_MODIFY_EXISTING_TRIDENT     Prevents any and all modification to the existing Trident installation (if any). Required if DISABLE_PROMPTS is true, otherwise defaults to false.
+
+  ----- General Configuration
+  *KUBECONFIG                        Path to the KUBECONFIG for the cluster you'd like to manage. Required.
+  COMPONENTS                         Determines what will be installed/upgraded. Default is ALL_ASTRA_CONTROL.
+  IMAGE_PULL_SECRET                  Image pull secret for the Docker registry.
+  NAMESPACE                          Overrides EVERY resource's namespace (for fresh installs only, not upgrades).
```

Comment:
What happens if NAMESPACE is set on an upgrade?

Reply:
Trident would stay where it is, while the connector gets installed in the given NAMESPACE. Kind of weird behavior to be honest, but we don't currently "detect" to check if a connector is already installed.

Reply:
would it install 2 connectors if the user entered a diff namespace than when initially installed?

Reply:
Yes, unfortunately. This is kind of a known issue that we deliberately planned on not fixing for this release since it's unlikely to happen I think, but it might be a good idea to add a note in the help description about it. I'll see what I can come up with.